README.md

```diff
@@ -1 +1,182 @@
+
+The following variables control some aspects of the script functionality. They can be set as job variables in the script or on the web pages. In the script, they need to be accessed as `${CUSTOM_ENV_<VARIABLE>}`.
+
+#### Global Options
+
+These variables are not SLURM specific and can be used in the default `ENROOT` only mode.
+
++ `CI_JOB_IMAGE` (YAML script `image:` option): a standard docker image for enroot to instantiate a container from;
++ `CI_WS`: a directory with shared data access across all nodes to be used as a workspace.
+
+Optional:
++ `USE_NAME`: instead of an automatically generated name, use a specific name for the container (and SLURM job if applicable). This name needs to be unique!
++ `NVIDIA_VISIBLE_DEVICES`: a value passed to the enroot container to control NVIDIA device visibility. When no GPU is available or used, `void` should be passed.
++ `CCACHE_DIR`: sets where the Ccache directory is mounted to in the enroot container when used.
++ `CCACHE_MAXSIZE`: sets a custom maximum limit to the Ccache directory size.
++ `KEEP_CONTAINER`: a non-zero value allows to not delete the container after usage, except if an error occurred.
+
+#### SLURM Behavior
+
+When any of these variables are set, instead of directly running the container on the node where `gitlab-runner` is running, this will submit a job instead. These variables allow to control the SLURM job submission and related behavior.
+
++ `SLURM_PARTITION`: the value of the SLURM `--partition` parameter, e.g., `gpu`.
++ `SLURM_EXCLUSIVE`: when non-zero, adds the SLURM `--exclusive` parameter.
++ `SLURM_TIME`: the value of the SLURM `--time` parameter, e.g. `00:30:00`.
++ `SLURM_GRES`: the value of the SLURM `--gres` parameter.
++ `USE_SLURM`: if no other variables are set, setting this enables SLURM mode of execution for this job.
+
+These variables control the SLURM job waiting loop behavior:
++ `SLURM_UPDATE_INTERVAL`: the sleeping time between two job status checks.
++ `SLURM_PENDING_LIMIT`: the job pending time limit, the default is 12 hours.
+
+## Installation
+The instructions are for a standard Linux system that already supports user mode GitLab and has enroot installed (see [dependencies](#Dependencies)). Also, refer to the [NHR@KIT CI user documentation][nhr-kit-cx] which detail this setup on their systems.
+
+### Installing a gitlab-runner
+
+The standard `gitlab-runner install` command can be used. Make sure to select the custom executor, see [gitlab runner install documentation][gitlab-runner-install]. Here is an example of what a runner configuration can look like:
+
+``` yaml
+[[runners]]
+  name = "enroot executor"
+  url = "https://gitlab.com"
+  token = "<token>"
+  executor = "custom"
+  builds_dir = "/workspace/scratch/my-ci-project/gitlab-runner/builds/"
+  cache_dir = "/workspack/scratch/my-ci-project/gitlab-runner/cache/"
+  environment = ["CI_WS=/workspace/scratch/my-ci-project/", "CCACHE_DIR=/ccache", "CCACHE_MAXSIZE=40G"]
```

Comment:
from the following description, the cache dir is `${CI_WS}${CCACHE_DIR}`
I would suggest to use `${CI_WS}/${CCACHE_DIR}`

Reply:
from the code, it seems to use the second path setting, so need to change here and doc

Reply:
Actually It's the other way around, this is the environment variable set inside the enroot container. The source of the ccache directory is always `${CI_WS}/ccache`, and in the container, it is mounted to `CCACHE_DIR`.